.github/workflows/pr_lint.yml

```diff
@@ -0,0 +1,40 @@
+name: PR Title Lint
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
```

Comment:
Lock down permissions: 

```
permissions:
  pull-requests: read
```

Reply:
(Still reviewing what the actual does)

Reply:
OK action is probably fine. Let's just restrict the GITHUB token permissions to pull-requests:read